src/ape_ethereum/__init__.py

```diff
@@ -20,5 +20,6 @@ def networks():
     for network_name, network_params in NETWORKS.items():
         yield "ethereum", network_name, create_network_type(*network_params)
 
-    # NOTE: This works for `geth --dev` as it gets chain_id from itself
+    # NOTE: This works for development environments, as they get chain_id from themselves
     yield "ethereum", "development", NetworkAPI
+    yield "ethereum", "mainnet-fork", NetworkAPI
```

Comment:
I believe `brownie` makes you add this via a CLI command.
Do we want to have that be how `mainnet-fork` gets added, or should we just add it here as a default, since it is rather common?

Reply:
You know, I was sort've hoping that the plugin can add extra networks... but I'm not sure about the security implications of that

this works too

Reply:
I tried to do that first! It was not showing up in the list for some reason, IDK.

Then I thought about ganache and how there could be multiple providers of mainnet-fork.

It seems like we either have to support adding custom networks or have mainnet-fork be a default network

Reply:
Fair. I guess this seems as good of an approach as any here

Reply:
Hmm..... Hardhat actually supports forking any of the testnets as well, I don't think we should limit it `mainnet`... Thinking!

Reply:
Should we just also add:

```
ethereum -> rinkeby-fork -> hardhat
ethereum -> infura-fork -> hardhat
ethereum -> kovan-fork -> hardhat
ethereum -> goerli-fork -> hardhat
```

?